### PR TITLE
Added GPS and HB data variables in their classes

### PR DESCRIPTION
**Pull request by ganeshram93**
1. Added getter functions for GPS and Heartbeat sensor and data
variables inside their respective classes
2. Changed variable name in tcp_server.py

Signed-off-by: ganeshram2493 <ganeshram2493@gmail.com>

### CODE REVIEW

File: HelloWorld_ST_Sensors/main.cpp [hunk 1]
```diff
@@ -152,16 +143,14 @@ int main() {
...
-    }
+    gps::gps_coordinates_s &gps_coordinates_data =
+        gps_obj.get_gps_coordinates();
+    debugPrintf("AP6H GPS:                        Latitude(deg): %f | "
+                "Longitude(deg): %f\r\n",
+                gps_coordinates_data.latitude, gps_coordinates_data.longitude);
 
     // Heartbeat Sensor
-    const heartbeat::bioData hb_data = heartbeat::get_hb_data();
+    const heartbeat::bioData hb_data = hb_obj.get_hb_data();
```
Comment (coder137) [inline HelloWorld_ST_Sensors/main.cpp]: const heartbeat::bioData `&` hb_data = hb_obj.get_hb_data();

File: HelloWorld_ST_Sensors/user/gps/gps.hpp [hunk 2]
```diff
@@ -23,31 +23,29 @@ enum GPRMC_NMEA_STRING_INDEX_DEFINITION {
...
+  gps_coordinates_s gps_data;
 
 public:
   adafruit_PA6H(PinName tx, PinName rx, uint32_t baud)
       : gps_uart(tx, rx, baud) {
     gps_uart.attach(callback(this, &adafruit_PA6H::gps_interrupt_cb));
   }
 
-  void register_func(void (*external_fn)(void));
+  gps_coordinates_s &get_gps_coordinates(void);
```
Comment (coder137) [inline HelloWorld_ST_Sensors/user/gps/gps.hpp]: `const gps_coordinates_s &`

File: HelloWorld_ST_Sensors/user/heartbeat/heartbeat.hpp [hunk 3]
```diff
@@ -194,9 +196,8 @@ class sparkfun_MAX32664 {
   const uint8_t SET_FORMAT = 0x00;
   const uint8_t ENABLE = 0x01;
   const uint8_t WRITE_SET_THRESHOLD = 0x01;
+  struct bioData body;
```
Comment (coder137) [inline HelloWorld_ST_Sensors/user/heartbeat/heartbeat.hpp]: no struct keyword

File: HelloWorld_ST_Sensors/user/internet/internet.cpp [hunk 4]
```diff
@@ -98,24 +101,34 @@ static void update_co_data(spec::CarbonMonoxide *co) {
...
 
+static void update_hb_data(heartbeat::sparkfun_MAX32664 *hb) {
+  heartbeat::bioData hb_data = hb->get_hb_data();
+  uint8_t buffer[8] = {0};
+  sprintf((char *)buffer, "%u,%u,", hb_data.heartRate, hb_data.oxygen);
+  strcat((char *)send_data, (const char *)buffer);
+}
+
+static void update_gps_data(gps::adafruit_PA6H *gps) {
+  gps::gps_coordinates_s gps_data = gps->get_gps_coordinates();
```
Comment (coder137) [inline HelloWorld_ST_Sensors/user/internet/internet.cpp]: const gps_coordinates_s &

File: HelloWorld_ST_Sensors/user/slave/i2c_slave.cpp [hunk 5]
```diff
@@ -273,13 +273,13 @@ void SlaveCommunication::send_heartbeat_data() {
 
 void SlaveCommunication::send_gps_coordinates_data() {
   uint8_t tx_data[ARR_SIZE * 2] = {0};
-  gps::gps_coordinates_s data = gps::get_gps_coordinates();
+  gps::gps_coordinates_s data = gps_.get_gps_coordinates();
```
Comment (coder137) [inline HelloWorld_ST_Sensors/user/slave/i2c_slave.cpp]: const gps_coordinates_s &

**Review by coder137 — CHANGES_REQUESTED** [review]

File: HelloWorld_ST_Sensors/user/slave/i2c_slave.cpp (hunk 5 above)
Comment (ganeshram93) [reply HelloWorld_ST_Sensors/user/slave/i2c_slave.cpp]: Updated in new PR.

File: HelloWorld_ST_Sensors/user/heartbeat/heartbeat.hpp (hunk 3 above)
Comment (ganeshram93) [reply HelloWorld_ST_Sensors/user/heartbeat/heartbeat.hpp]: Updated in new PR.

File: HelloWorld_ST_Sensors/user/gps/gps.hpp (hunk 2 above)
Comment (ganeshram93) [reply HelloWorld_ST_Sensors/user/gps/gps.hpp]: Updated in new PR.

File: HelloWorld_ST_Sensors/main.cpp (hunk 1 above)
Comment (ganeshram93) [reply HelloWorld_ST_Sensors/main.cpp]: Updated in new PR.

**Review by coder137 — APPROVED** [review]

**Pull request merged**